src/main/java/frc/robot/Robot.java

```diff
@@ -44,22 +45,13 @@ public void robotInit() {
     // yet. Thus, their requires() statements may grab null pointers. Bad
     // news. Don't move it.
     this.m_oi = new OI();
+    new Thread(()-> {
```

Comment:
Could we move this into the USBCamera's constructor? This keeps the camera code all together.

If not, I'd rather make a function in USCCamera class called "cameraInit()" which calls this piece of code.